CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.16.0 (2024-10-30)
+
+### BREAKING CHANGES
+* concurrent worker agent processes (#160) ([`9f7da7f`](https://github.com/aws-deadline/deadline-cloud-test-fixtures/commit/9f7da7f6873f0f25c7f67b34de7f1e28b85dc947))
```

Comment:
mmm, because it loses the context, I don't think this really explains what is changing

Reply:
I've revised the changelog to add a separate breaking change section with details for consumers